proto/pdpb.proto

```diff
@@ -53,6 +53,8 @@ service PD {
 
     rpc ScanRegions(ScanRegionsRequest) returns (ScanRegionsResponse) {}
 
+    rpc BatchScanRegions(BatchScanRegionsRequest) returns (BatchScanRegionsResponse) {}
```

Comment:
Will it replace ScanRegions?

Reply:
Yes, I think `ScanRegions` can be deprecated then.

Reply:
Then how about adding a comment for it?

Reply:
Done.